packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 1.4.0
+
+- New version of SuiteCloud Unit Testing that provides SuiteScript 2.x stubs for all SuiteScript modules.
+- Show the active project name its authentication ID when working with multiple projects.
```

Comment:
Sorry for not specifying it before, but this feature doesn't depend on multiple projects being opened. You could have just one project opened and it will show the project name and its authentication ID.
It's true that its actually very useful when working with multiple projects though, I don't know if we could rephrase it.
Sorry for the late reply.

Reply:
Changed.